.github/ISSUE_TEMPLATE/request-for-position.yml

```
@@ -0,0 +1,82 @@
+name: Request for position
+description: Solicit WebKit's position on an emerging web specification
+body:
+  - type: markdown
+    attributes:
+      value: "## WebKittens who can provide input"
+  - type: input
+    id: webkittens
+    attributes:
```

Forms inputs can have labels. Might look better than the H2s
```suggestion
    attributes:
      label: WebKittens who can provide input
```